libtiledbvcf/src/write/writer.cc

```diff
@@ -429,6 +431,12 @@ std::pair<uint64_t, uint64_t> Writer::ingest_samples(
             throw std::runtime_error(
                 "Error submitting TileDB write query; unexpected query "
                 "status.");
+
+          if (ingestion_params_.verbose)
```

Comment:
Definitely a fan of more info for the user during ingestion but this will be printing quite a bit of text. Any change it could negatively impact performance?